developer/src/kmc-model/src/build-trie.ts

```diff
@@ -32,7 +32,7 @@ export function createTrieDataStructure(filenames: string[], searchTermToKey?: (
   filenames.forEach(filename => parseWordListFromFilename(wordlist, filename));
 
   let trie = buildTrie(wordlist, searchTermToKey as SearchTermToKey);
-  return JSON.stringify(trie);
+  return `{"data":${JSON.stringify(trie.compress())},"totalWeight":${trie.getTotalWeight()}}`;
```

Comment:
A thought that just occurred to me:  if we wanted to support a "debug-mode" Trie, we could to this by preserving the original output from `buildTrie` here - the deleted lines 219-222 below.

I'll leave that call for later, though... just thought it might be worth the mention.

Reply:
```suggestion
  return JSON.stringify({
    data: trie.compress(), 
    totalWeight: trie.getTotalWeight()
  });
```

Is this the same? Without hacking together JSON with string builder?